packages/governance/test/unitTests/test-paramGovernance.js

```diff
@@ -223,10 +223,9 @@ test('change multiple params', async t => {
 
   /** @type {GovernedPublicFacet<unknown>} */
   const publicFacet = await E(governorFacets.creatorFacet).getPublicFacet();
-  const notifier = makeNotifierFromAsyncIterable(
-    await E(publicFacet).getSubscription(),
-  );
-  const update1 = await notifier.getUpdateSince();
+  const governorSubscription = await E(publicFacet).getSubscription();
+  const governorUpdates = await E(governorSubscription)[Symbol.asyncIterator]();
```

Comment:
I'm not clear on why such use of `makeNotifierFromAsyncIterable` for testing in governance and run-protocol causes lockup (e.g., https://github.com/Agoric/agoric-sdk/runs/7138160731?check_suite_focus=true : "_unitTests › paramGovernance › change multiple params … Error: Promise returned by test never resolved_"). It could be related to use of a fake timer, or could be an actual bug. But the tests don't appear to have been expecting lossyness, so I feel mostly fine about direct use of `await … next()`.